worlds/stardew_valley/items.py

```diff
@@ -148,29 +106,11 @@ def load_item_csv():
             id = int(item["id"]) if item["id"] else None
             classification = ItemClassification[item["classification"]]
             groups = {Group[group] for group in item["groups"].split(",") if group}
-            items.append(ItemData(id, item["name"], classification, groups))
+            mod_name = str(item["mod_name"]) if item["mod_name"] else None
```

Comment:
```suggestion
            mod_name = item["mod_name"]
```
based on your dataclass definition, this should be sufficient.

Reply:
This way of writing it will create empty strings for vanilla items, but the Design was to use "None" for vanilla, so I believe this won't work properly